controllers/front/ExpressCheckout.php

```diff
@@ -189,7 +189,16 @@ private function createCustomer($email, $firstName, $lastName)
         $customer->email = $email;
         $customer->firstname = $firstName;
         $customer->lastname = $lastName;
-        $customer->passwd = md5(time() . _COOKIE_KEY_);
+
+        if (class_exists('PrestaShop\PrestaShop\Core\Crypto\Hashing')) {
+            $crypto = new PrestaShop\PrestaShop\Core\Crypto\Hashing();
+            $customer->passwd = $crypto->hash(
```

Comment:
I see that the password is hashed 2 times here, maybe it should be like this?  $crypto->hash(time(), _COOKIE_KEY_)

Reply:
That's right